specs/app_settings/01_main.rst

```diff
@@ -0,0 +1,225 @@
+    - Option to rate the app in the store.
+    - View app version (For bug reporting)
+    - View used open source licenses
+
+
+Inputs
+-----------
+
+ - Depending on the use case: Password and fingerprint, Terms, Privacy Policy
+
+
+Outputs
+----------
+
+Outputs depend on the selected option.
+- Open webpage (Terms & Privacy Policy)
+- Start change password flow
+- Fingerprint has been enabled / disabled
+- Open up email program (Give feedback)
+- Open store for rating / writing review (Rate app)
+- App version is displayed
+- View used open source licenses
+
+
+Use Case Scenarios
+-----------------------
+
+App version
+~~~~~~~~~~~~~~~~
+
+Preconditions
++++++++++++++
+
+- App is set up with a password
+
+Steps
++++++
+
+1. Open app settings
+
+Postconditions
+++++++++++++++
+
+- User can see app version in way so we know which version they are using.
+
+
+View Terms of Use
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Inherits from `App version`_
+
+Steps
++++++
+
+2. Open Terms of Use
+3. Browser opens with https://safe.gnosis.io/terms
+4. User goes back
+
+Postconditions
+++++++++++++++
+
+- User is back on the settings page.
+
+
+View Privacy Policy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Inherits from `View Terms of Use`_
+
+Steps
++++++
+
+2. Open Privacy Policy
+3. Browser opens with https://safe.gnosis.io/privacy
+4. User goes back
+
+
+Start change password flow
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Inherits from `App version`_
+
+Steps
++++++
+
+2. Open "Change password"
+
+Postconditions
+++++++++++++++
+
+- "Change password" flow is started
+
+
+Enable finterprint
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Preconditions
++++++++++++++
+
+- App is set up with a password
+- Finterprint is disabled
+
+Steps
++++++
+
+1. Open app settings
+2. Disable finterprint
+
+Postconditions
+++++++++++++++
+
+- Fingerprint is disabled.
+- User is still on settings page.
+
+
+Disable finterprint
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Preconditions
++++++++++++++
+
+- App is set up with a password
+- Finterprint is enabled
+
+Steps
++++++
+
+1. Open app settings
+2. Enable finterprint
+3. User has to confirm with fingerprint
+
+Postconditions
+++++++++++++++
+
+- Fingerprint is enabled.
+- User is still on settings page.
+
+
+Rate app
+~~~~~~~~~~~~~
+
+Inherits from `App version`_
+
+Steps
++++++
+
+2. Open "Rate app"
+
+Postconditions
+++++++++++++++
+
+- Store is opened on a screen where user can rate and review the Gnosis Safe.
+
+
+Give feedback
+~~~~~~~~~~~~~~~
+
+Inherits from `View Terms of Use`_
+
+Steps
++++++
+
+2. Open "Give feedback"
+3. Email program opens with a prefilled email with the following text:
+
+ App version: <iOS/Android> - <app_version>
+
+ Safe addresses:
+     0xFirstSafeAddress
+
+     0xSecondSafeAddress
+
+ Feedback:
+
+4. Enter text
+5. Send email
+
+
+View open source licenses
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Inherits from `View Terms of Use`_
+
+Steps
++++++
+
+2. Open "Licenses"
+3. Browser opens with https://safe.gnosis.io/licenses
```

Comment:
This side doesn't exist yet, right?

Do we use the same for Android and iOS?

Reply:
Yes, doesn't exist. I would put both on 1 page, yes. And then perhaps link to the respective subsection, i.e. e.g.

```
https://safe.gnosis.io/licenses#ios
https://safe.gnosis.io/licenses#android
```

Wanted to get Michel's input on this idea.